packages/launcher/test/unit/windows_spec.ts

```diff
@@ -149,23 +159,74 @@ describe('windows browser detection', () => {
 
   context('#getPathData', () => {
     it('returns path and browserKey given path with browser key', () => {
-      const res = windowsHelper.getPathData('C:\\foo\\bar.exe:firefox')
+      const browserPath = 'C:\\foo\\bar.exe'
+      const res = windowsHelper.getPathData(`${browserPath}:firefox`)
 
-      expect(res.path).to.eq('C:\\foo\\bar.exe')
+      expect(res.path).to.eq(doubleEscape(browserPath))
+      expect(res.browserKey).to.eq('firefox')
+    })
+
+    it('returns path and browserKey given path with a lot of slashes plus browser key', () => {
+      const browserPath = 'C:\\\\\\\\foo\\\\\\bar.exe'
+      const res = windowsHelper.getPathData(`${browserPath}:firefox`)
+
+      expect(res.path).to.eq(doubleEscape(browserPath))
+      expect(res.browserKey).to.eq('firefox')
+    })
+
+    it('returns path and browserKey given nix path with browser key', () => {
+      const browserPath = 'C:/foo/bar.exe'
+      const res = windowsHelper.getPathData(`${browserPath}:firefox`)
+
+      expect(res.path).to.eq(doubleEscape(browserPath))
       expect(res.browserKey).to.eq('firefox')
     })
 
     it('returns path and chrome given just path', () => {
-      const res = windowsHelper.getPathData('C:\\foo\\bar\\chrome.exe')
+      const browserPath = 'C:\\foo\\bar\\chrome.exe'
+      const res = windowsHelper.getPathData(browserPath)
+
+      expect(res.path).to.eq(doubleEscape(browserPath))
+      expect(res.browserKey).to.eq('chrome')
+    })
 
-      expect(res.path).to.eq('C:\\foo\\bar\\chrome.exe')
+    it('returns path and chrome given just nix path', () => {
+      const browserPath = 'C:/foo/bar/chrome.exe'
+      const res = windowsHelper.getPathData(browserPath)
+
+      expect(res.path).to.eq(doubleEscape(browserPath))
       expect(res.browserKey).to.eq('chrome')
     })
 
+    it('returns path and edge given just path', () => {
```

Comment:
Could also use a test for msedge.exe - that is what #14716 is about

Reply:
Yes, updated